src/components/layouts/pages/results/Results.js

```diff
@@ -51,13 +53,15 @@ function Results({ jobStatus, taskStatus, onBackClick }) {
 Results.propTypes = {
     jobStatus: PropTypes.oneOf(_.values(JOB_STATUS)).isRequired,
     taskStatus: PropTypes.oneOf(_.values(TASK_STATUS)),
+    isCompliant: PropTypes.bool,
     onBackClick: PropTypes.func.isRequired,
 };
 
 function mapStateToProps(state) {
     return {
         jobStatus: getJobStatus(state),
         taskStatus: getTaskStatus(state),
+        isCompliant: isCompliant(state),
```

Comment:
Let's rename prop to simple `compliant`, selector will remain with `is` prefix. Also let's mark it as required in prop types.